src/metabase/models/permissions.clj

```diff
@@ -853,6 +927,41 @@
   (delete-related-permissions! group-or-id (apply (partial feature-perms-path :download :full) path-components))
   (delete-related-permissions! group-or-id (apply (partial feature-perms-path :download :limited) path-components)))
 
+(let [delete (fn delete [s to-delete] (str/replace s to-delete ""))
```

Comment:
FWIW I usually prefer `let ... fn` to `letfn` since it makes it easier to capture the function in a var and call it directly in the REPL when debugging